packages/auth/src/authClients/supabase.ts

```diff
@@ -6,9 +6,17 @@ export type Supabase = SupabaseClient
 export type SupabaseUser = User
 
 export interface AuthClientSupabase extends AuthClient {
+  /**
+   * Log in an existing user, or login via a third-party provider.
+   * @param option The user login details.
+   * @param option.email The user's email address.
+   * @param option.password The user's password.
+   * @param option.provider One of the providers supported by GoTrue.
```

Comment:
Maybe you could use literal union types here? 
Something like
```suggestion
   * @param {'google'|'facebook'} option.provider One of the providers supported by GoTrue.
```

Reply:
https://jsdoc.app/tags-type.html

Reply:
Added `{'bitbucket' | 'github' | 'gitlab' | 'google' | 'azure'}` from https://github.com/supabase/gotrue-js/blob/7580c34aeb68d5e5b8d3548e0615801b7a9472d1/src/lib/types.ts#L1.

Yes, Azure is implemented -- just not yet configurable in the Supabase Admin Dashboard.